app/controllers/catalog_controller.rb

```diff
@@ -25,6 +25,7 @@ class CatalogController < ApplicationController
 
     # items to show per page, each number in the array represent another option to choose from.
     # config.per_page = [10,20,50,100]
+    config.per_page = [25]
```

Comment:
This change means the only per-page option available to users will be 25?

Reply:
It means the default will be 25 instead of 10. As now, the user will not have the option of selecting the results per page.